### ISSUE TITLE
[issue] Page not flipping properly

### ISSUE THREAD

**Issue by enikolas** [darenju/react-flip-page#24]
## How to reproduce

Create a project from scratch using create-react-app:

```
npx create-react-app reproducing-issue
cd reproducing-issue
```

Install you dependency from npm:

```
npm i -P react-flip-page@0.10.1
```

And, inside ```src/App.js```, change the content to be:

```
import React, { Component } from 'react';
import FlipPage from 'react-flip-page';
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="App">
        <FlipPage>
          <article>
            <h1>My awesome first article</h1>
            <p>My awesome first content</p>
          </article>
          <article>
            <h1>My wonderful second article</h1>
            <p>My wonderful second content</p>
          </article>
          <article>
            <h1>My excellent third article</h1>
            <p>My excellent third content</p>
          </article>
        </FlipPage>
      </div>
    );
  }
}

export default App;
```

Now, run the application with:

```
npm start
```

And try to navigate until page 3.

## What is happening

It navigates through page 1 and 2, but suddenly, it moves back to page 1, not allowing you to see page 3.

## Expected behavior

It should be able to flip accross all the pages, in the same way as it is showed in the [examples page](http://darenju.me/react-flip-page/examples)

**Comment by darenju** [comment]
This is very weird, I am able to reproduce the issue.

I don't know what could cause this, I will try to work on it.

Thank you for reporting.

**Comment by darenju** [comment]
Apparently, this is due to `react-flip-page` being compiled for `react@15`. Pushing the react version to 16 and re-compiling works.

I am performing those changes and updating the repository.

**Comment by darenju** [comment]
@enikolas Can you please confirm the issue is fixed with `react-flip-page@0.10.3`?

I have tried it and it's working, but one more confirm would be cool.

**Comment by enikolas** [comment]
@darenju It worked! Well done.

Cheers mate!
